frontend/src/components/ui/ExecutionTable.jsx

```diff
@@ -11,9 +11,12 @@ import {
 import { PipelineStopButton } from "./PipelineStopButton";
 import { useAtom } from "jotai";
 import { activeConfigurationAtom } from "@/atoms/anvilConfigurationsAtom";
+import { useImmerAtom } from "jotai-immer";
+import { workspaceAtom } from "@/atoms/pipelineAtom";
 
 export const ExecutionTable = ({ executions, closeModal }) => {
   const [configuration] = useAtom(activeConfigurationAtom);
+  const [, setWorkspace] = useImmerAtom(workspaceAtom);
   const selectExecution = (execution) => {
     const key = execution.pipeline + "." + execution.id;
 
```

Comment:
Extra attention needed here, the `setWorkspace` was undefined. I think this would fix a bug for an unimplemented feature, but I am not sure if it's still needed.

Reply:
This is good to add. This code isn't called at the moment but this component is usable if we want to present executions in a table view